src/plugins/maps_ems/tsconfig.json

```diff
@@ -7,5 +7,8 @@
     "declarationMap": true
   },
   "include": ["common/**/*", "public/**/*", "server/**/*", "./config.ts"],
-  "references": [{ "path": "../../core/tsconfig.json" }]
+  "references": [
+    { "path": "../../core/tsconfig.json" },
+    { "path": "../../../x-pack/plugins/licensing/tsconfig.json" }
```

Comment:
Can src/plugin depend on x-pack/plugin?

Reply:
yes. We can introduce optional plugin dependencies from `src/*` on `x-pack/plugin`.